services/worker-manager/README.md

```diff
@@ -2,10 +2,16 @@
 
 The worker manager service manages workers, including interacting with cloud services to create new workers on demand.
 
-It currently includes providers for:
+## Providers
 
-* Static Workers
-* Google Cloud
+The service configuration defines a number of providers, indexed by `providerId`, along with configuration.
+Each provider has a `providerType` indicating the class that implements the provider.
+
+This service currently includes providers for:
+
+* Static Workers (`static`)
```

Comment:
@djmitche How will these be mapped onto `providerId`s? For example, if I create a workerpool with `providerId` "banana" - how do we know to which cloud we should send the tasks? (I am working on the worker pool creation ui, and started having this question)

Reply:
My understanding of our plan is that we should have an endpoint on worker-manager that lists providers. It will return data about which implementation a provider uses. Does that make sense?